packages/openapi3/src/openapi.ts

```diff
@@ -1540,6 +1540,22 @@ function createOAPIEmitter(
     }
   }
 
+  function validateComponentFixedFieldKey(type: Type, name: string) {
+    // eslint-disable-next-line no-useless-escape
+    const pattern = /^[a-zA-Z0-9\.\-_]+$/;
```

Comment:
oh, yes, we don't need `\.`, `/^[a-zA-Z0-9.\-_]+$/` is enough, please use this one, also update the doc. LGTM if it's fixed